src/Network/Send/kRO/RagexeRE_2015_05_13a.pm

```diff
@@ -0,0 +1,151 @@
+#########################################################################
+#  OpenKore - Packet sending
+#  This module contains functions for sending packets to the server.
+#
+#  This software is open source, licensed under the GNU General Public
+#  License, version 2.
+#  Basically, this means that you're allowed to modify and distribute
+#  this software. However, if you distribute modified versions, you MUST
+#  also distribute the source code.
+#  See http://www.gnu.org/licenses/gpl.html for the full license.
+########################################################################
+package Network::Send::kRO::RagexeRE_2015_05_13a;
+
+use strict;
+use base qw(Network::Send::kRO::RagexeRE_2014_10_22b);
+
+sub new {
+my ($class) = @_;
+my $self = $class->SUPER::new(@_);
+
+	my %packets = (
+	'0958' => ['item_take', 'a4', [qw(ID)]],
+	'0885' => ['item_drop', 'v2', [qw(index amount)]],
+	'0879' => ['storage_item_add', 'v V', [qw(index amount)]],
+	'0864' => ['storage_item_remove', 'v V', [qw(index amount)]],
+	#'0438' => ['skill_use_location', 'v4', [qw(lv skillID x y)]],
```

Comment:
I'm so sorry because at first, i had compared with 2013_08_07a that uses

 '0438' => ['skill_use_location', 'v4', [qw(lv skillID x y)]],#10

Then in the 2014_X_22 is different, and on 2013 it's equal. 
And well, thx